pkg/controller/storagecluster/reconcile.go

```diff
@@ -254,7 +254,7 @@ func (r *ReconcileStorageCluster) reconcileNodeTopologyMap(sc *ocsv1.StorageClus
 		return err
 	}
 
-	if sc.Status.NodeTopologies == nil {
+	if sc.Status.NodeTopologies == nil || sc.Status.NodeTopologies.Labels == nil {
```

Comment:
There's nothing wrong with this change, but i'm unsure if it potentially fixes anything.

looking at the code, it doesn't look possible for `Labels==nil` to occur. When NodeTopologies is defined, the Label map is also created.

Reply:
oh i see what's going on now. That Labels map can get removed when the StorageCluster is updated if it is empty because of our usage of `omitempty` in the json definition of that value.